src/corehost/cli/args.cpp

```diff
@@ -80,7 +80,10 @@ bool parse_arguments(const pal::string_t& deps_path, const pal::string_t& probe_
     pal::string_t opts_deps_file = _X("--depsfile");
     pal::string_t opts_probe_path = _X("--additionalprobingpath");
     pal::string_t deps_file = opts.count(opts_deps_file) ? opts[opts_deps_file] : deps_path;
-    pal::string_t probe_path = opts.count(opts_probe_path) ? opts[opts_probe_path] : probe_dir;
+    if (opts.count(opts_probe_path))
+    {
+        args.probe_paths.insert(opts[opts_probe_path]);
```

Comment:
Does this allow multiple probe paths to be added to the arg list? It does not seem so.